src/components/FeatureFlagModal/FeatureFlagModal.tsx

```diff
@@ -304,6 +305,12 @@ export default function FeatureFlagModal() {
           featureFlag={FeatureFlag.progressIndicatorV2}
           label="Refreshed swap progress indicator"
         />
+        <FeatureFlagOption
+          variant={BaseVariant}
+          value={useAndroidGALaunchFlag()}
```

Comment:
such a weird pattern we have going on here, you'd think we'd have something like `useFlag(Flags.AndroidGALaunch)` and then just a simpler component here like `<FeatureFlag from={Flags.AndroidGALaunch} />` or something along those lines... not saying to change it here but something to look at in the future

Reply:
yeeaah feature flags needs improvement for sure, kinda on my backlog but i have noted your comments https://linear.app/uniswap/issue/WEB-2932/feature-flag-code-improvements